src/server.ts

```diff
@@ -49,8 +84,16 @@ app.get('/api/credentials/:service', async (request, response) => {
 });
 
 app.get('/api/credentials', async (_req, res) => {
```

Comment:
Please use one naming scheme, so either req, res or request, response.
Also remove the underscore, as you are using the parameter
```suggestion
app.get('/api/credentials', async (req, res) => {
```